packages/extension_google_sign_in_as_googleapis_auth/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.0.0
```

Comment:
Why is this a breaking change? Nothing should affect clients in a breaking way.

Reply:
This used to be 2.0.1 until I changed the version of googleapis_auth to 1.1.0 for the new import.